pcp/PCPDynamicScreen.c

```diff
@@ -28,16 +28,19 @@ in the source distribution for its full text.
 
 void PCPDynamicScreens_appendDynamicColumns(PCPDynamicScreens* screens, PCPDynamicColumns* columns) {
    for (unsigned int i = 0; i < screens->count; i++) {
-      PCPDynamicScreen *screen = Hashtable_get(screens->table, i);
+      PCPDynamicScreen* screen = Hashtable_get(screens->table, i);
       if (!screen)
          return;
 
-      for (unsigned int column = 0; column < screen->totalColumns; column++) {
-         screen->columns[column].id = columns->offset + columns->cursor;
+      for (unsigned int j = 0; j < screen->totalColumns; j++) {
+         PCPDynamicColumn* column = &screen->columns[j];
+
+         column->id = columns->offset + columns->cursor;
          columns->cursor++;
```

Comment:
Is this cursor still necessary when we index over the array instead of the cursor?